frontend/src/reducers/authReducer.tsx

```diff
@@ -0,0 +1,25 @@
+import { AuthActionType, AuthActions, IMessageResponse  } from '../actions/authActions'
+
+interface IAuthState {
+    passwordRecoveryResponse?: IMessageResponse
+}
+
+export const initialState = {
+    passwordRecoveryResponse: {
+        message: '',
+        error: false
+    }
+}
+
+const authReducer = (state: IAuthState = initialState, action: AuthActions ): IAuthState => {
```

Comment:
In general, I don't think we need redux state here to be used. Purpose of redux state is to be a GLOBAL state - keep the data that will be shared across whole application. For instance: information about user that currently logged in.
Usually we want to put in redux state those data which will be shown in several places/pages/components. 
All other data should stored as where it used or something like that. Error message and state of recovery page are only used on this recovery page.
So, it's it about using redux.

But if you want just to practice redux to get more familiar with - it's ok, feel free write code in a such way (but please do not do it in real world :) ) 
Just warn me that "here I just wanna play with redux and not really care about other stuff")

Reply:
I didn't want to use redux, it was a way to get response from a dispatched action (response of the backend on a request that was done in actions). Is there another way to get it? (except for performing the request from the component)

Reply:
I optimized the code according to all of your previous suggestions. But I still kept the redux state because I don't know how to get the server response inside my component otherwise